wgpu-hal/src/gles/web.rs

```diff
@@ -147,22 +147,33 @@ impl crate::Instance<super::Api> for Instance {
         _display_handle: raw_window_handle::RawDisplayHandle,
         window_handle: raw_window_handle::RawWindowHandle,
     ) -> Result<Surface, crate::InstanceError> {
-        if let raw_window_handle::RawWindowHandle::Web(handle) = window_handle {
-            let canvas: web_sys::HtmlCanvasElement = web_sys::window()
+        let canvas: web_sys::HtmlCanvasElement = match window_handle {
```

Comment:
Don't we need to do this for WebGPU as well?

Reply:
Indeed!
Done as well.